corgi/core/models.py

```diff
@@ -653,7 +648,9 @@ def provides_queryset(self, using: str = "read_only") -> QuerySet["Component"]:
         return (
             Component.objects.filter(pk__in=unique_provides)
             # Remove .exclude() below when CORGI-428 is resolved
-            .exclude(type=Component.Type.GOLANG, name__contains="./").using(using)
+            .exclude(type=Component.Type.GOLANG, name__contains="./")
+            .external_components()
```

Comment:
I originally tried it that way, but it didn't work. I think that's because that query only select root_components. Whereas the package we are trying to filter, is included in the manifest as a result of being included in this provides_queryset.